pilot/pkg/serviceregistry/kube/controller.go

```diff
@@ -156,25 +137,18 @@ func (c *Controller) notify(obj interface{}, event model.Event) error {
 	return nil
 }
 
-// createInformer registers handlers for a specific event.
+// createCacheHandler registers handlers for a specific event.
 // Current implementation queues the events in queue.go, and the handler is run with
 // some throttling.
 // Used for Service, Endpoint, Node and Pod.
 // See config/kube for CRD events.
 // See config/ingress for Ingress objects
-func (c *Controller) createInformer(
-	o runtime.Object,
+func (c *Controller) createCacheHandler(
```

Comment:
nit: now that this isn't so long, can you bring this function declaration back onto one line pleaes

Reply:
make sense

Reply:
@ZackButcher addressed